diff --git a/node_modules/downshift/dist/downshift.esm.js b/node_modules/downshift/dist/downshift.esm.js
index 94d8e20..c1ebb8f 100644
--- a/node_modules/downshift/dist/downshift.esm.js
+++ b/node_modules/downshift/dist/downshift.esm.js
@@ -3420,7 +3420,9 @@ function useCombobox(userProps) {
 
         var latestState = latest.current.state;
 
-        if (latestState.isOpen) {
+        // The app is relying on the old behavior, "broken" by https://github.com/downshift-js/downshift/pull/1150
+        // Enter key should submit the form if there is no selection in the dropdown.
+        if (latestState.isOpen && latestState.highlightedIndex > -1) {
           event.preventDefault();
         }
 
